addons/web_editor/static/src/xml/editor.xml

```diff
@@ -5,9 +5,9 @@
     <!--=================-->
 
     <!-- Editor toolbar -->
-    <t t-name="web_editor.toolbar">
+    <t t-name="web_editor.toolbar" owl="1">
         <div id="toolbar" class="oe-toolbar oe-floating">
-            <div id="style" class="btn-group dropdown">
+            <div t-if="props.showStyle" id="style" t-attf-class="btn-group {{ props.dropDirection }}">
```

Comment:
Wait, those options are actually new features, aren't they ? We didn't have those before.